src/db/addStudentMgmt.sql

```diff
@@ -303,10 +310,17 @@ RETURNS TABLE("FirstName" VARCHAR(50),
              )
 AS
 $$
-BEGIN
-   RAISE WARNING 'Function not implemented';
-END
-$$ LANGUAGE plpgsql
+
+SELECT COALESCE(s.fname, ''),
+       COALESCE(s.mname, ''), 
+       COALESCE(s.lname, ''), 
+       schoolissuedid, email, year
+FROM student s
+WHERE TRIM(COALESCE(s.fname)) ILIKE TRIM($1) AND
```

Comment:
These `COALESCE` statements on lines 319-321 don't seem to be doing much. I'm guessing that they should have `''` as their second parameter?

Reply:
You are correct.  Intent was to account for a student without a middle name, etc.  Will be addressed in a commit, shortly.